en/index.md

```diff
@@ -362,6 +362,51 @@ state of each unit and service:
 ...
 ```
 
+### Placeholders in code blocks
+
+There are many times when it will be necessary for the user to supply terms or
+data to a command which cannot be known in advance, for example IP addresses or
+names. It is common to substitute such values with a placeholder, where the
+placeholder consists of delimiters that contain terms that suggest the
+necessary input. For example:
+
+```
+lxc delete <instance_name>/<snapshot_name>
+```
+
+Here, the reader is expected to substitute their values for the placeholders.
+To minimise the risk of errors, instruct the reader that such values need to be
+substituted, especially when the first placeholder is referenced.
+
+This approach is in common use and is effective for short commands, but is less
+suitable for a long block of code where the user then has to edit a multi-line
+command and search through to find the placeholders to replace. A useful
+approach in these circumstances is to adjust the code to define user-supplied
+data as environmental variables. For example:
+
+````
+Set the desired channel:
+
+```
+CHANNEL=1.30/stable
```

Comment:
Should your example include a placeholder instead of a real value?
```suggestion
CHANNEL=<channel_name>
```

Reply:
yes. The idea is that the text would explain what the user needs to do. In this case a sensible real value is there. I prefer this approach because:
- if the reader blindy copies everything it still works
- it is also suitable for automated testing

Reply:
I mean of course, no. the example is correct. I'm right.